apis/infix-en_GB/atrium-api-infix-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/chronoZonedDateTimeAssertions.kt

```diff
@@ -74,3 +74,73 @@ infix fun <T : ChronoZonedDateTime<out ChronoLocalDate>> Expect<T>.isAfterOrEqua
 infix fun <T : ChronoZonedDateTime<out ChronoLocalDate>> Expect<T>.isEqual(
     expected: ChronoZonedDateTime<*>
 ): Expect<T> = _logicAppend { isEqual(expected) }
+
+/**
+ * Expects that the subject of the assertion (a [ChronoZonedDateTime])
+ * is before or equals the [expected] [ChronoZonedDateTime] given as [String].
+ * Check [ch.tutteli.atrium.logic.impl.DefaultChronoZonedDateTimeAssertions.parseZonedDateTime] to see which values are allowed for [expected].
```

Comment:
I think we should write the supported formats here. I doubt someone wants to go and have a look at an internal method.
Also, we can then write that ZoneDesignator like [Europe/Zurich] is only supported experimentally.

Reply:
Nice work in api-fluent, can you please copy it to api-infix as well